server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/LongKeyedBucketOrds.java

```diff
@@ -101,6 +106,45 @@ private LongKeyedBucketOrds() {}
      */
     public abstract BucketOrdsEnum ordsEnum(long owningBucketOrd);
 
+    public Iterator<Long> keyOrderedIterator(long owningBucketOrd) {
```

Comment:
I wonder if `PrimitiveIterator.OfLong` is better. I think it's about the same to implement it.

Reply:
And could you add javadocs? I am still trying to digest what is going on here :)

Reply:
yeah, sorry, I should have marked this "draft".  Was really just pushing it for some early feedback and CI testing.  I'll add java doc today.